project/src/js/ui/round/socketHandler.js

```diff
@@ -63,7 +63,7 @@ export default function(ctrl, onFeatured, onUserTVRedirect) {
       if (ctrl.data.game.speed === 'correspondence') {
         removeOfflineGameData(ctrl.data.url.round.substr(1));
       }
-      if (!ctrl.data.player.spectator) {
+      if (ctrl.data.tournament || !ctrl.data.player.spectator) {
```

Comment:
I have removed. I could see being able to return to the tournament with a
popup button being marginally useful, but acknowledge that there is a big
back button right there at the top of the page and don't feel strongly.

On Mon, May 9, 2016 at 4:40 AM, Vincent Velociter <notifications@github.com>
wrote:

> In project/src/js/ui/round/socketHandler.js
> <https://github.com/veloce/lichobile/pull/327#discussion_r62465593>:
>
> > @@ -63,7 +63,7 @@ export default function(ctrl, onFeatured, onUserTVRedirect) {
> >        if (ctrl.data.game.speed === 'correspondence') {
> >          removeOfflineGameData(ctrl.data.url.round.substr(1));
> >        }
> > -      if (!ctrl.data.player.spectator) {
> > +      if (ctrl.data.tournament || !ctrl.data.player.spectator) {
>
> I'd remove this. Not useful to show menu if spectator of a tournament.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/veloce/lichobile/pull/327/files/d0652a502378c4b86249942bc653a9abb8c5a8ad#r62465593>
>